src/instrument/devices/flag_4.py

```diff
@@ -36,8 +36,8 @@ def __init__(
     flag_4 = FCpt(EpicsMotor, "{motor_prefix}:{_flag_4_motor}", labels={"motors"})
 
 
-fl4 = Flag4(
-    name="fl4",
+flag4 = Flag4(
```

Comment:
This creates a motor object `flag4.flag_4`.  To move it, the user might use `yield from bps.mvr(flag4.flag_4, -0.05)`.  Is there any advantage to encapsulating this motor inside a device?

Reply:
fixed